src/components/attendee/VideoComms/internal/VideoCommsControls.tsx

```diff
@@ -33,11 +35,14 @@ export const VideoCommsControls: React.FC<VideoCommsControlsProps> = ({
       {startAudio &&
         (audioEnabled ? (
           <span onClick={stopAudio}>
-            <FontAwesomeIcon icon={faVolumeUp} />
+            <FontAwesomeIcon icon={faMicrophone} />
```

Comment:
Based on the logic, it shows whether the person hears people, not if they hear him. Are you sure about the `microphone` icon here?